e2e/client/playwright/hightlights.spec.ts

```diff
@@ -0,0 +1,174 @@
+
+        // create fresh article
+        await monitoring.selectDeskOrWorkspace('Sports');
+        await monitoring.createArticleFromTemplate('story', {slugline: 'article 1'});
+        await page.locator(s('authoring-topbar', 'save')).click();
+
+        await monitoring.executeActionOnMonitoringItem(
+            page.locator(s('article-item=article 1')),
+            'Mark for highlight',
+            'Highlight 1',
+        );
+
+        // create package highlights
+        await page.locator(s('workspace-navigation')).getByRole('button', {name: 'Highlights'}).hover();
+        await page.locator(s('workspace-navigation')).getByRole('button', {name: 'Highlight 1'}).click();
+        await page.locator(s('articles-list--toolbar')).getByRole('button', {name: 'Create'}).click();
+        await expect(page.locator(s('authoring', 'package-items=article 1'))).toBeVisible();
+        await page.locator(s('authoring', 'package-title')).fill('Package Highlight 2');
+        await page.locator(s('authoring-topbar')).getByRole('button', {name: 'Save'}).click();
+        await expect(
+            page.locator(s('authoring-topbar')).getByRole('button', {name: 'Save'}).locator(s('loading-indicator')),
+        ).toBeVisible();
+        await expect(
+            page.locator(s('authoring-topbar')).getByRole('button', {name: 'Save'}).locator(s('loading-indicator')),
+        ).not.toBeVisible();
+        await page.locator(s('authoring-topbar', 'close')).click();
+
+        // check visibility of highlight
+        await page.goto('/#/workspace/monitoring');
+        await expect(page.locator(s('monitoring-view', 'article-item=Package Highlight 2'))).toBeVisible();
+    });
+
+    test('publishing highlights package', async ({page}) => {
+        const monitoring = new Monitoring(page);
+
+        await restoreDatabaseSnapshot();
+        await page.goto('/#/workspace/monitoring');
+
+        await monitoring.selectDeskOrWorkspace('Sports');
+        await expect(
+            page.locator(s('monitoring-group=Sports / Working Stage', 'article-item=Package Highlight 1')),
+        ).toBeVisible();
+        await expect(
+            page.locator(s('monitoring-group=Sports desk output', 'article-item=Package Highlight 1')),
+        ).not.toBeVisible();
+
+        await monitoring.executeActionOnMonitoringItem(
+            page.locator(s('article-item=Package Highlight 1')),
+            'Edit',
+        );
+
+        await page.locator(s('authoring-topbar', 'open-send-publish-pane')).click();
+        await page.locator(s('interactive-actions-panel', 'publish')).click();
+
+        await expect(
+            page.locator(s('monitoring-group=Sports / Working Stage', 'article-item=Package Highlight 1')),
+        ).not.toBeVisible();
+        await expect(
+            page.locator(s('monitoring-group=Sports desk output', 'article-item=Package Highlight 1')),
+        ).toBeVisible();
+    });
+
+    test('exporting highlights', async ({page}) => {
```

Comment:
are you exporting only one or multiple? if one, then `test exporting a highlight`